python/lsst/meas/algorithms/maskStreaks.py

```diff
@@ -45,6 +47,8 @@ class Line:
     rho: float
     theta: float
     sigma: float = 0
+    chi2: float = 0
+    modelMaximum: float = 0
```

Comment:
Please add a description of these two attributes somewhere in the class doc string